Python/Tests/TestData/Cookiecutter/template_opentxt/cookiecutter.json

```diff
@@ -0,0 +1,10 @@
+{
+  "project_name": "Default Project Name",
+  "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
+  "_visual_studio_post_cmds": [
+    {
+      "name": "File.OpenFile",
+      "args": "\"{{cookiecutter._output_folder_path}}\\readme.txt\""
```

Comment:
I'd rather `args` be a list rather than having to quote each argument. I expect most to be a single argument though, so perhaps treat a string as one argument and a list as multiple? `ProcessOutput.QuoteSingleArgument` is your friend when going to a single string (though passing `string[]` as a variant should be fine too).